### ISSUE TITLE
✨ Improve freshdesk templates

### ISSUE THREAD

**Issue by tristan-orourke** [GCTC-NTGC/gc-digital-talent#11375]
## ✨ Feature

Attempt to fix all a11y errors with Freshdesk found in:

- https://github.com/GCTC-NTGC/gc-digital-talent/issues/10971

## 🕵️ Details

The following is a list of all errors we uncovered. If they are not checked, we could not fix them. Results will be documented.

- [x] No skip link
- [x] Missing roles (headings)
- [ ] Not user scalable
- [x] Empty link content
- [ ] Missing form label

### Skip Link

There was a skip link but it was hidden even when focused so it seemed like there wasn't. I removed the class doing this and now it only appears while focused:

![Image](https://github.com/user-attachments/assets/621e1692-433d-4002-a06c-31ff8fc95eb0)

### Missing roles

Was able to apply the proper values. However, I suggest we look into producing better markup. We can do this, I was just proving the point we could fix this for now.

### User scaling

We can add the proper tag to the head to make it user scalable but the original tag remains. I am submitting a ticket to see if we can remove the original tag.

#### Response from Freshdesk

> With regards to removing the original viewport meta tag, unfortunately this would not be possible since the meta tags are added by default in the backend. 

### Empty link text

Looks like this was resolved without the need for intervention.

![Image](https://github.com/user-attachments/assets/16dfe05e-cd48-4f8c-ab28-495c663d00bb)

### Form label

There does not seem to be a setting for this so I submitted a request along with the user scalable item.

#### Freshdesk response 

> Upon checking internally, currently there is no way to assign the placeholder as a text label on the customer portal. While I understand from an accessibility perspective for users, I shall have this conveyed as a feature enhancement request to our product team.
> However, please note that we ​do not have an ETA as to when the feature requests would be picked up for development, as the prioritization of such requests are taken care of by our product team.

![Image](https://github.com/user-attachments/assets/ec614e76-15f1-4e4c-815f-909d1e2cf7fe)

## 📋 Results

We seem to have pretty good control over the theme and I have a local copy to edit all templates. I will be okay if we need to just duplicate the viewport tag (not ideal but w/e). However, I'd like to see the ticket form have a label on the type.

## 🙋‍♀️ Proposed Implementation

The the editor to tweak the existing theme as needed.

## ✅ Acceptance Criteria

- [ ] All a11y issues fixed where possible
- [ ] Any issues that could not be fixed are documented

**Comment by esizer** [comment]
Leaving this here so I don't lose it.

https://support.freshdesk.com/support/solutions/folders/117110
